### DISCUSSION TITLE
How to set the default "Debugger to launch" & related "Local path" via premake

### DISCUSSION THREAD

**Discussion by danalemaytelescope** [premake/premake-core#2079] [Q&A]
Greetings,

Looking to find how to set the Debugger to launch & the local path under Debugging in the project settings in VS. 
Any help is appreciated.

**Comment by Jarod42** [comment]
There is [`debugdir`](https://premake.github.io/docs/debugdir/) for the later.
I don't think there is something to select which debugger to use.
